helm/idol-nifi/values.yaml

```
@@ -161,10 +161,12 @@ nifi:
   # -- What proportion of the pod memory to allocate for JVM usage
   jvmMemoryRatio: 0.5
 
-  # -- the flowfile definition to import. 
-  # Set this to any non-existing path to bypass flow import (start with a blank flow).
-  # Customize the NiFi image and set this to a filepath within that image to import your own customized flow.
-  flowfile: /scripts/flow-basic-idol.json
+  # -- The flow definitions to import into NiFi registry. Specify the file, registry bucket name (will be created if not found), and whether to import the flow as a process group into NiFi.
+  # Customize the NiFi image and set this to one or more filepaths within that image to import your own customized flows.
```

```suggestion
  # Customized flows can be added via a custom NiFi image or mounted into the pod (see `additionalVolumes` and `additionalVolumeMounts`)
```
Suggested comment update to make it clear you don't need a custom nifi image to get a flow in